### PR TITLE
[wallet connect] add executeSerializedMoveCall

### PR DESCRIPTION
**Pull request by gegaowp**
Add executeSerializedMoveCall to unblock capybara DApp,
the added function header is 
```
public executeSerializedMoveCall(txBytes: Uint8Array)
```

It now has the option from the Dapp console
Screen Shot 2022-07-07 at 12 47 40 PM

### CODE REVIEW

**Review by pchrysochoidis — COMMENTED** [review]
Looks good, if you have some time to revert [the renaming of transaction to tx](https://github.com/MystenLabs/sui/pull/3072/files#r916609922) and consider [this](https://github.com/MystenLabs/sui/pull/3026#discussion_r916596104) instead of introducing the txBytes field let me know, otherwise I will approve it as is

File: wallet/src/dapp-interface/DAppInterface.ts [hunk 1]
```diff
@@ -86,11 +86,21 @@ export class DAppInterface {
         );
     }
 
-    public executeMoveCall(transaction: MoveCallTransaction) {
+    public executeMoveCall(tx: MoveCallTransaction) {
```
Comment (pchrysochoidis) [inline wallet/src/dapp-interface/DAppInterface.ts:89]: Why did we rename this? Leaving it as `transaction` we can avoid changing 

```
{
                type: 'execute-transaction-request',
                transaction,
}
```
to

```
{
                type: 'execute-transaction-request',
                transaction: tx,
}
```

**Comment by gegaowp** [comment]
addressed comments 1. revert the transaction naming change 2. change TransactionRequest type definition per Pavlos's comment
Also added contents to show on signing page for serialized txn.

**Review by pchrysochoidis — APPROVED** [review]
Thanks @gegaowp for addressing the comments. Although, I thinκ we don't need to introduce the `txBytes` and use the `type` everywhere from the messages to handling. But this looks good as is and we can update it later. 👍

**Pull request merged**